integration-tests/utils/integration-utils.js

```diff
@@ -165,6 +165,7 @@ const label = ({ labelContains }) => {
 
 const fillIn = async (page, xpath, text) => {
   const input = await page.waitForXPath(xpath, defaultToVisibleTrue());
+  await delay(1000);
```

Comment:
I can reduce this if it seems overlong. I was finding Puppeteer was losing the first 2-8 chars of the input without a delay here.

Reply:
Please add a comment about why the delay is here. I might also suggest reducing to more like 500ms.

Note also that the type method below has a delay property. I think that is the delay between typing characters through, and not an initial delay.